app/assets/stylesheets/styleguide/_typography.scss

```diff
@@ -1,218 +0,0 @@
-@import "_print.scss";
-
```

Comment:
I've remove the only reference to `styleguide/_typography.scss`. There are no Sass compile errors.